internal/services/monitor/monitor_scheduled_query_rules_alert_resource.go

```diff
@@ -92,6 +92,12 @@ func resourceMonitorScheduledQueryRulesAlert() *pluginsdk.Resource {
 				Required:     true,
 				ValidateFunc: azure.ValidateResourceID,
 			},
+			"auto_mitigation_enabled": {
+				Type:          pluginsdk.TypeBool,
+				Optional:      true,
+				Default:       true,
```

Comment:
Set default to false here, otherwise when throttling is specified instead, autoMitigate is still always sent along as true in the REST call
```suggestion
				Default:       false
```